components/navigation.css

```diff
@@ -0,0 +1,113 @@
+.navigation p {
+  font-size: 3rem;
+}
+.navbar {
+  width: 100%;
+  height: 20rem;
+  background-color: rgb(249, 250, 243);
+  overflow-y: scroll;
+  border: 1px solid black;
+  position: relative;
+}
+.navbar p {
+  font-size: 1rem;
+  padding: 2rem;
+}
+.nav-header {
+  height: 5rem;
+  background-color: var(--primary-color);
+  display: flex;
+  justify-content: space-between;
+}
+.nav-logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 1rem;
+}
+.nav-logo p {
+  font-size: 2rem;
+  font-weight: bold;
+  margin-left: 1rem;
+}
+.nav-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.nav-icon p {
+  font-size: 20px;
```

Comment:
Ya, I have used it everywhere but here I was not getting the appropriate font size using rem so that's why here I have used px.. but thanks for noticing that I will keep this in mind.